### PR TITLE
update to 1.19.3

### PR DESCRIPTION
**Pull request by MoriyaShiine**
Most of the changes are just gradle/json updates. Moved registering out of the mod initializer and into the same place where other attributes are registered (EntityAttributes) because the registry is not frozen there.

### CODE REVIEW

File: src/main/java/dev/emi/stepheightentityattribute/mixin/EntityAttributesMixin.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+import dev.emi.stepheightentityattribute.StepHeightEntityAttributeMain;
+import net.minecraft.entity.attribute.ClampedEntityAttribute;
+import net.minecraft.entity.attribute.EntityAttribute;
+import net.minecraft.entity.attribute.EntityAttributes;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+
+@Mixin(EntityAttributes.class)
+public abstract class EntityAttributesMixin {
```
Comment (emilyploszaj) [inline src/main/java/dev/emi/stepheightentityattribute/mixin/EntityAttributesMixin.java:11]: Why was this moved to a mixing, can you no longer register normally?

File: src/main/java/dev/emi/stepheightentityattribute/mixin/EntityMixin.java [hunk 2]
```diff
@@ -12,7 +12,7 @@
 @Mixin(Entity.class)
 public abstract class EntityMixin {
 
-	@Redirect(at = @At(value = "FIELD", target = "Lnet/minecraft/entity/Entity;stepHeight:F", opcode = Opcodes.GETFIELD), method = "adjustMovementForCollisions")
+	@Redirect(at = @At(value = "FIELD", target = "Lnet/minecraft/entity/Entity;stepHeight:F", opcode = Opcodes.GETFIELD), method = "adjustMovementForCollisions*")
```
Comment (emilyploszaj) [inline src/main/java/dev/emi/stepheightentityattribute/mixin/EntityMixin.java]: What is this asterisk?

File: src/main/java/dev/emi/stepheightentityattribute/mixin/EntityAttributesMixin.java (hunk 1 above)
Comment (MoriyaShiine) [reply src/main/java/dev/emi/stepheightentityattribute/mixin/EntityAttributesMixin.java:11]: Yep. It yelled at me for the registry being frozen while it was still in the mod initializer, so I moved it to the same place where the others are registered

File: src/main/java/dev/emi/stepheightentityattribute/mixin/EntityMixin.java (hunk 2 above)
Comment (MoriyaShiine) [reply src/main/java/dev/emi/stepheightentityattribute/mixin/EntityMixin.java]: It may or may not just be an mc mod dev plugin thing, it stops it from warning about multiple methods with different parameters. I'm unsure if it's actually part of mixin, but it's functionally identical to not having it

Comment (emilyploszaj) [reply src/main/java/dev/emi/stepheightentityattribute/mixin/EntityMixin.java]: After looking into it it might match undesirable methods with different names in the future? I'd prefer mcdev just be allowed to complain.

Comment (MoriyaShiine) [reply src/main/java/dev/emi/stepheightentityattribute/mixin/EntityMixin.java]: Done
